src/GitHubCreateMergePRs/config.xml

```diff
@@ -15,7 +15,6 @@
 
     <!-- Roslyn vs-deps branches (flowing between releases) -->
     <merge from="release/dev16.10-vs-deps" to="release/dev16.11-vs-deps" />
-    <merge from="release/dev16.11-vs-deps" to="release/dev17.0-preview1-vs-deps" />      
```

Comment:
dev16.11-vs-deps changes won't flow into `main-vs-deps` - should this just be changed from `release/dev17.0-preview1-vs-deps` to `main-vs-deps`